docs/Features/Prompt Syntax.md

```diff
@@ -123,6 +123,7 @@
             - You can do this all in one prompt to individual refine specific faces separately
             - Without this, if there are multiple people, it will do a bulk segmented refine on all faces combined
             - Note the index order is sorted from leftmost detection to right
+        - To filter specific classes, append `:class_ids` where `class_ids` is a comma-separated list of class IDs, e.g., `yolo-modelnamehere:0,1,2`
```

Comment:
Er, this doesn't work.
Segment has a dynamic comma separated list of numbers as its input already. It needs a separator mark of some form. Whether that's `model:0:1:2,0.8,1` or `model:0,1,2:0.8,1`